### PR TITLE
Add generators to scaffold application

### PR DESCRIPTION
**Pull request by EvgeniyRRU**

### CODE REVIEW

File: lib/crud_responder/locales/en.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+en:
```
Comment (olegantonyan) [inline lib/crud_responder/locales/en.yml]: Usual place for locales is `config/locales` and there are few already. No need to create a new one

**Comment by olegantonyan** [comment]
Nice work! Thx

File: lib/crud_responder/locales/en.yml (hunk 1 above)
Comment (EvgeniyRRU) [inline lib/crud_responder/locales/en.yml]: It's will copy to config/locales in rails app. But can you clarify me what do you mean?

Comment (olegantonyan) [inline lib/crud_responder/locales/en.yml]: The gem already has locales `cd /tmp && git clone git@github.com:olegantonyan/crud_responder.git && ls crud_responder/config/locales`

**Comment by EvgeniyRRU** [comment]
Fixed it.

**Pull request merged**